feed-page/README.md

```diff
@@ -1 +1,13 @@
-# Feed Page
\ No newline at end of file
+# Feed Page
+
+
+
+```json
+{
+   "$type":"feedPage@1", // Type definition with spec version number (1)
+   "userId":"d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd", // This is the global userId used by SkyID, not the app-specific public key (To ensure interoperability)
```

Comment:
`userId` is ok but I miss a reference to the `publicKey` of the user. Having only the SkyID `userId` is like saying "you must use SkyID or you need to know how SkyID is generating the userId, there is no other way to access the data of a specific user".

Reply:
SkyID manages all cross-skapp data like username and profile picture, so it makes sense to refer to that. A developer only needs to look up the `profile` datakey of the `userId` once to get the sub-app public key. If you really need the app-specific public key of a user directly, you also already know it because you literally used it to retrieve this feedPage document. :smile:  This does not enforce the usage of SkyID because we will also define the profile data structure in this repo which then must be used by SkyID and all other auth solutions